tools/mo/openvino/tools/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -189,6 +189,13 @@ def groupconv_to_conv(op: Node):
         # We use add_destination method here to support case with multiple destinations of source port
         weights_node.in_port(0).get_source().get_connection().add_destination(op.in_port(1))
         weights_node.in_port(0).disconnect()
+    elif weights_node.type == 'Convert':
```

Comment:
please explicitly check this case. Other cases should be covered by else below:
`elif weights_node.type == 'Convert' and weights_node.dst_type == np.float16 and weights_node.in_port(0).get_source().node.type == `Const``

Reply:
Done.